examples/tutorial/x/blog/query_test.go

```diff
@@ -6,7 +6,7 @@ import (
 	"github.com/iov-one/weave"
 	"github.com/iov-one/weave/store"
 	"github.com/iov-one/weave/weavetest"
-	"github.com/iov-one/weave/weavetest/assert"
+	"github.com/stretchr/testify/assert"
```

Comment:
I believe this is an error.
The other file is having the same issue.

Reply:
Not really, I tried to migrate as many as possible testify requires to `weavetest.assert` via Find and Replace. Some of these occurrences had incompatible signatures so I had to resort to importing `testify.assert` as part of migration, as it used to use `require`. 

It makes no difference in the context of tests and their results. But if you insist - I can look into cherry-picking this stuff and reverting it.

Reply:
Basically it's a step towards `unify tests`, it did help get rid of testify in some places, but not all :(